templates/2_create_tiled_mibi_run.ipynb

```diff
@@ -13,12 +13,14 @@
    "cell_type": "markdown",
```

Comment:
<p>Content in the GIFs is good. Both of them are a bit too slow. </p><p> </p><p>Second GIF should have an example of selecting two FOVs, de-selecting one of them, then actually clicking the delete button</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>